src/store.ts

```diff
@@ -2,48 +2,107 @@ import Vue from 'vue';
 import Vuex, { StoreOptions } from 'vuex';
 import {ParsedRpcRequest} from '@/lib/RequestTypes';
 import {KeyInfo} from '@/lib/KeyInfo';
-import {State as RpcState} from '@nimiq/rpc';
+import {State as RpcState, State} from '@nimiq/rpc';
 import {KeyStore} from '@/lib/KeyStore';
 import {RpcResult as KeyguardResult} from '@/lib/keyguard/RequestTypes';
+import { AddressInfo } from '@/lib/AddressInfo';
 
 Vue.use(Vuex);
 
 export interface RootState {
-    request?: ParsedRpcRequest;
+    request: ParsedRpcRequest | null;
     rpcState: RpcState | null;
     keys: KeyInfo[]; // TODO: this is not JSON compatible, is this a problem?
     keyguardResult: KeyguardResult | Error | null;
+    activeLoginId: string | null;
+    activeAccountPath: string | null;
 }
 
 const store: StoreOptions<RootState> = {
-    state: {
-        rpcState: null, // undefined is not reactive
-        keys: [],
-        keyguardResult: null, // undefined is not reactive
-    },
+    state: (() => {
```

Comment:
Is that the recommended way of doing that?
I would have rather put that into the `created` method similar to the initKeys.